### PR TITLE
[zh-cn]: update the translation of <menu>

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests
part of https://github.com/mdn/translated-content/issues/14272

### CODE REVIEW

File: files/zh-cn/web/html/element/menu/index.md [hunk 1]
```diff
@@ -1,156 +1,140 @@
...
+          ><a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/list_role"
+            >list</a
+          ></code
+        >
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">允许的 ARIA 角色</th>
+      <td>
+        <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/directory_role"><code>directory</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/group_role"><code>group</code></a>、<code><a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/listbox_role">listbox</a></code>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/menu_role"><code>menu</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/menubar_role"><code>menubar</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/none_role"><code>none</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/presentation_role"><code>presentation</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/radiogroup_role"><code>radiogroup</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/tablist_role"><code>tablist</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/toolbar_role"><code>toolbar</code></a> 或 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/tree_role"><code>tree</code></a>
```
Comment (T34-active) [inline files/zh-cn/web/html/element/menu/index.md:129]:
```suggestion
        <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/directory_role"><code>directory</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/group_role"><code>group</code></a>、<code><a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/listbox_role">listbox</a></code>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/menu_role"><code>menu</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/menubar_role"><code>menubar</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/none_role"><code>none</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/presentation_role"><code>presentation</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/radiogroup_role"><code>radiogroup</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/tablist_role"><code>tablist</code></a>、<a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/toolbar_role"><code>toolbar</code></a> 或 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/tree_role"><code>tree</code></a>。
```

File: files/zh-cn/web/html/element/menu/index.md [hunk 2]
```diff
@@ -1,156 +1,140 @@
...
+      <th scope="row">
+        <a href="/zh-CN/docs/Web/HTML/Content_categories"
+          >内容分类</a
+        >
+      </th>
+      <td>
+        <p>
+          <a href="/zh-CN/docs/Web/HTML/Content_categories#流式内容"
+            >流式内容</a
+          >，如果元素的子元素包含至少一个
```
Comment (yin1999) [inline files/zh-cn/web/html/element/menu/index.md:88]:
```suggestion
          >。如果元素的子元素包含至少一个
```

**Review by yin1999 — APPROVED** [review]
Thank you @T34-active

**Comment by T34-active** [comment]
> Thank you @T34-active

我太开心了 终于不是说我格式问题了

**Comment by yin1999** [comment]
> > Thank you @T34-active
> 
> 我太开心了 终于不是说我格式问题了

Great job :tada:

**Pull request merged**